### PR TITLE
2.3.6

### PR DESCRIPTION
**Pull request by es128**
opening as a PR just to make it easier to evaluate per #47

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 sudo: false
 language: node_js
 node_js:
-  - "0.10"
+  - "stable"
```
Comment (es128) [inline .travis.yml]: I think it's worth adding `4` here also as the LTS version

**Comment by es128** [comment]
Gotta run, but based on 60 seconds of skimming, seems fine. I can take a closer look later.

File: .travis.yml (hunk 1 above)
Comment (jonschlinkert) [inline .travis.yml]: good one, I keep forgetting to add that to my `update-travis` template

**Comment by jonschlinkert** [comment]
k, thx. no rush

**Comment by jonschlinkert** [comment]
@es128 going to go ahead and merge so we can get the lazy-cache stuff merged in

**Pull request merged**